packages/table/examples/tableSortableExample.tsx

```diff
@@ -103,7 +103,7 @@ class RankSortableColumn extends AbstractSortableColumn {
     }
 }
 
-class RecordSortableColumn extends AbstractSortableColumn {
+export class RecordSortableColumn extends AbstractSortableColumn {
```

Comment:
This had to do with better type inference in TS 2.2 -- the `public state` of TableSortableExample was inferred as using private types, so tsc wanted me to make these types public. I refactored a bit in 09772d4 so that we could expose fewer types.